005-rbac/view-pods.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  creationTimestamp: null
```

Comment:
The `creationTimestamp` keys are not needed. Pls delete

Reply:
Dun dun dun!